resources/prosody-plugins/mod_muc_system_chat_message.lua

```diff
@@ -0,0 +1,140 @@
+-- A global module which can be used as http endpoint to send system chat messages to meeting participants. The provided token
+--- in the request is verified whether it has the right to do so.
+-- Copyright (C) 2024-present 8x8, Inc.
+
+-- curl https://{host}/send-system-message  -d '{"message": "testmessage", "to": "{connection_jid}", "room": "{room_jid}"}' -H "content-type: application/json" -H "authorization: Bearer {token}"
+
+module:set_global();
```

Comment:
Is there a reason this to be global, as you are listening only for the muc component. You can be loading just there.